common/stats/stats_names.go

```diff
@@ -0,0 +1,81 @@
+package stats
+
+/*
+This file defines all the metrics being collected.   As new metrics are added please follow this pattern.
+ */
+
+const (
+	/*** Node availability (ClusterManger) metrics ***/
+	/*
+	the number of worker nodes that are available or running tasks (not suspended)
+	 */
+	ClusterAvailableNodes = "availableNodes"
+
+	/*
+	the number of idle worker nodes (available, but not running a task)
+	 */
+	ClusterIdleNodes ="idleNodes"
+
+	/*
+	the number of lost worker nodes (not responding to status requests)
+	 */
+	ClusterLostNodes ="lostNodes"
+
+
+	/*** Scheduler Metrics ***/
+	/*
+	The number of jobs in the inProgress list at the end of each time through the
+	scheduler's job handling loop
+	 */
+
+	SchedAcceptedJobsGauge="schedAcceptedJobsGauge"
+
+	/*
+	the number of tasks that have finished (including those that have been killed)
+	 */
+	SchedCompletedTaskCounter = "completedTaskCounter"
```

Comment:
Do we want to remove the 'sched' prefix from all of these strings? It should already have 'scheduler' and whatever reason I had to include 'sched' prefix may no longer apply.